tests/client/core/components/TestInstallButton.js

```diff
@@ -155,4 +161,53 @@ describe('<InstallButton />', () => {
     assert.equal(buttonComponent.type, Button);
     assert.strictEqual(buttonComponent.props.disabled, true);
   });
+
+  it('renders a button for OpenSearch regardless of mozAddonManager', () => {
+    const root = render({
+      addon: {
+        ...fakeAddon,
+        hasAddonManager: true,
+        type: ADDON_TYPE_OPENSEARCH,
+      },
+    });
+
+    assert.equal(root.type, 'div');
+    const buttonComponent = root.props.children[1];
+    assert.equal(buttonComponent.type, Button);
+    assert.equal(buttonComponent.props.className,
+      'Button InstallButton-button');
+    assert.equal(buttonComponent.props.children, 'Add to Firefox');
+  });
+
+  it('disables the OpenSearch button if not compatible', () => {
+    const root = render({
+      addon: { ...fakeAddon, type: ADDON_TYPE_OPENSEARCH },
+      getClientCompatibility: getClientCompatibilityFalseOpenSearch,
+    });
+
+    assert.equal(root.type, 'div');
+    const buttonComponent = root.props.children[1];
+    assert.equal(buttonComponent.type, Button);
+    assert.equal(buttonComponent.props.className,
```

Comment:
could this just be `assert.include(buttonComponent.props.className, 'InstallButton-button--disabled')` ?